docs/source/index.rst

```diff
@@ -0,0 +1,53 @@
+=================================
+Welcome to wfsim's documentation!
+=================================
+
+Github page: https://github.com/XENONnT/WFSim
+
+**WFSim** is the wavefrom simulation framework for XENONnT, built on top of the `straxen framework <https://github.com/XENONnT/straxen>`_.
+
+Installation
+==================
+
+1. To get started for simulating XENONnT please follow the `straxen setup guide <https://straxen.readthedocs.io/en/latest/setup.html>`_. Simulating XENON1T works out of the box.
+
+2. On top of ``straxen``, install ``wfsim``:
+
+.. code-block:: bash
+
+   pip install wfsim
+
+
+.. wfsim documentation master file, created by
+sphinx-quickstart on Sat Feb 27 10:43:35 2021.
+You can adapt this file completely to your liking, but it should at least
+contain the root `toctree` directive.
+
+
+.. toctree::
+    :maxdepth: 1
+    :caption: Contents:
+
+    intro
+
+.. toctree::
+    :maxdepth: 2
+    :caption: Tutorials:
+
+    tutorials/Understanding_WFSim_P1.ipynb
+    tutorials/Understanding_WFSim_P2.ipynb
+    tutorials/Advanced_tricks.ipynb
```

Comment:
Can you remove this guy and replace it with Fax_tutorial.ipynb

This advanced_tricks guy is secretly very outdated